share/spack/qa/check_dependencies

```diff
@@ -65,6 +65,32 @@ for dep in "$@"; do
 
         exit 1
     fi
+
+    # Flake8 requires setuptools in order to run.
+    # Otherwise, it prints out this error message:
+    #
+    #   Traceback (most recent call last):
+    #     File: "/usr/bin/flake8", line 5, in <module>
+    #       from pkg_resources import load_entry_point
+    #   ImportError: No module named pkg_resources
```

Comment:
I believe the same problem also occurs for Sphinx. @citibeth had this problem because py-sphinx wasn't activated. I'm not sure exactly which packages it requires at runtime though.